### PR TITLE
Update ringpop-go to use updated bark import path

### PR DESCRIPTION
**Pull request by prashantv**
bark has moved from github.com/uber to github.com/uber-common

r: @akshayjshah

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
 	"ImportPath": "github.com/uber/ringpop-go",
 	"GoVersion": "go1.5.1",
+	"Packages": [
+		"./..."
```
Comment (CorgiMan) [inline Godeps/Godeps.json]: What's this?

Comment (prashantv) [inline Godeps/Godeps.json]: When you use `godeps`, you should save dependencies for all subpackages as well, and so you should run:
```bash
godep save ./...
```

It looks like it was previously only doing godeps for the root package.

Comment (CorgiMan) [inline Godeps/Godeps.json]: Nice, good to know!

**Comment by CorgiMan** [comment]
Let's get the Money!

**Pull request merged**